app/controllers/static_controller.rb

```diff
@@ -0,0 +1,2 @@
+class StaticController < ApplicationController
+end
```

Comment:
- [ ] It would be better to create the index method for the static controller. Since the root routes points to `static#index`